pages/hydra/developer_resources/admin-sets.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Make an Admin Set"
```

Comment:
This is about "User Roles" not admin sets.  "AdminSets" are defined in CurationConcerns/Sufia and Hyrax. It's important that we don't confuse these concepts: See: https://github.com/projecthydra/curation_concerns/blob/master/app/models/admin_set.rb